installscripts/jazz-terraform-unix-noinstances/scripts/DeleteStackPlatformServices.py

```diff
@@ -58,7 +58,7 @@ def deleteCloudFormationService(service_name):
 stackName = sys.argv[1].lower() + "-"
 deleteClientServices = sys.argv[2]
 
-platformServices = ['cognito-authorizer', 'logs', 'usermanagement', 'services-handler', 'events', 'services', 'logout', 'login', 'cloud-logs-streamer', 'is-service-available', 'delete-serverless-service', 'create-serverless-service', 'email', 'events-handler', 'environments', 'scm-webhook' , 'environment-event-handler', 'deployments', 'deployments-event-handler']
+platformServices = ['cognito-authorizer', 'logs', 'usermanagement', 'services-handler', 'events', 'services', 'logout', 'login', 'cloud-logs-streamer', 'is-service-available', 'delete-serverless-service', 'create-serverless-service', 'email', 'events-handler', 'environments', 'scm-webhook' , 'environment-event-handler', 'deployments', 'deployments-event-handler', 'jazz_asset-event-handler']
```

Comment:
I believe jazz_ prefix should not be there

Reply:
Updated